frappe/core/doctype/role/role.py

```diff
@@ -31,7 +31,7 @@ def get_emails_from_role(role):
 
 	for user in users:
 		user_email, enabled = frappe.db.get_value("User", user.parent, ["email", "enabled"])
-		if enabled:
+		if enabled and user_email not in ["admin@example.com", "guest@example.com"]:
```

Comment:
Shouldn't this fixed at the root?

Reply:
We cant create admin and guest without email address as its a mandatory field, so we use placeholders instead.